Utilities/Dockers/VCFEval/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.10
```

Comment:
it there a need to keep [the rtg dockerfile](https://github.com/broadinstitute/palantir-workflows/blob/main/Utilities/Dockers/RTG/Dockerfile) around, or can this replace it?

Reply:
I think it depends on how we want to organize the dockers in the repo overall. Originally, it started from the philosophy of trying to have a bunch of lightweight docker images with "just" what you need for various tasks. But since often WDLs end up with tasks that combine lots of tools together, it might make sense to also add some of these beefier images to our repertoire as well, or even as replacement. 

I don't have strong opinions either way, other than I have been considering making a large "all-in-one" image for people who don't want to micromanage their environments, in which case this could be included in.